lib/ui/chat/widgets/message_widget.dart

```diff
@@ -84,6 +88,8 @@ class MessageWidget extends StatelessWidget {
         return IntrinsicWidth(
           child: Container(
             constraints: BoxConstraints(
+              // FIX: Maximum width constraint has been restored to its original state.
```

Comment:
[sug] Personally I’d remove this kind of comment. They’re easy to get outdated, and I’d rather only keep comments for things that are really important or hard to understand from the code itself. If you think this detail is worth keeping, maybe rephrase it to explain why the constraint matters (e.g., “Without max width, balloons don’t resize correctly”).